api/tests/Api/MaterialItems/UpdateMaterialItemTest.php

```diff
@@ -407,10 +407,7 @@ public function testPatchMaterialItemDoesNotCrashForLargeNumberForQuantity() {
             ]
         );
 
-        $this->assertResponseStatusCodeSame(200);
-        $this->assertJsonContains([
-            'quantity' => 0,
-        ]);
+        $this->assertResponseStatusCodeSame(400);
```

Comment:
This is now the behavior after php upgrade (not 100% sure why), but in my opinion this is now the better behavior than before